paymentapi/src/main/java/fi/hel/verkkokauppa/payment/util/RefundUtil.java

```diff
@@ -0,0 +1,17 @@
+package fi.hel.verkkokauppa.payment.util;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+
+
+public class RefundUtil {
```

Comment:
This is the exact same logic that is in PaymentUtil.generatePaymentOrderNumber(...) and thus does not follow "DRY" principles. I suggest we rename the id generation method to generatePaymentOrderNumber() -> generateIdWithTimestamp() and then leverage it in the refund flow.

Reply:
Good point, I will do that